main/acle.rst

```diff
@@ -404,6 +408,131 @@ Scalable Vector Extensions (SVE)
+|                                                                                |
+|      svint32_t svset_neonq[_s32](svint32_t vec, int32x4_t subvec)              |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svint64_t svset_neonq[_s64](svint64_t vec, int64x2_t subvec)              |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint8_t svset_neonq[_u8](svuint8_t vec, uint8x16_t subvec)              |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint16_t svset_neonq[_u16](svuint16_t vec, uint16x8_t subvec)           |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint32_t svset_neonq[_u32](svuint32_t vec, uint32x4_t subvec)           |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint64_t svset_neonq[_u64](svuint64_t vec, uint64x2_t subvec)           |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svfloat16_t svset_neonq[_f16](svfloat16_t vec, float16x8_t subvec)        |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svfloat32_t svset_neonq[_f32](svfloat32_t vec, float32x4_t subvec)        |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svfloat64_t svset_neonq[_f64](svfloat64_t vec, float64x2_t subvec)        |
++--------------------------------------------------------------------------------+
```

Comment:
Can you also add instances of both get and set for the `poly8x16_t` , `poly16x8_t` , `poly64x2_t` and `poly128_t` types?

Reply:
Any suggestions for the `poly128_t` signature? There's no `svint128_t`

Reply:
I didn't notice that the polynomial types don't exist anymore! I would guess probably the corresponding unsigned types should be used (to match the `pmull{b,t}` intrinsics), with `poly128_t` corresponding to `svuint64x2_t` based on the signature of e.g. `svpmullb_pair`? Would prefer a more authoritative opinion though, @fpetrogalli?

Reply:
Personally, I think we should only provide functions for element types that exist in both Neon and SVE.  The user can reinterpret the Neon type where necessary.

Reply:
I've decided to not to include polynomial types for now

Reply:
May I ask you to add this explanation in the design document?

```
Why polynomial type are not supported
---------------------

We only provide functions for element types that exist in both Neon and SVE.
The user can reinterpret the Neon type where necessary.

```